client/coral-admin/src/routes/Community/components/styles.css

```diff
@@ -240,3 +240,97 @@
   }
 }
 
+/* ========================================================================== */
+/* TABLE */
+/* ========================================================================== */
```

Comment:
`styles.css` is deprecated, we shouldn't add more styles to it.

Reply:
Adding a comment then.